### ISSUE TITLE
Cannot import sdk after gulp dist

### ISSUE THREAD

**Issue by DevJS** [lucascosta/facebook-js-ads-sdk#2]
Hi there,

I am encountering a problem with this npm package since the last update. 

When I use the package after a simple `npm install --save facebook-ads-sdk`, it works fine.
I wanted to make some modifications and I modified some files. For a debugging test purpose I just added a `console.log('hey')` in the method "getNodePath()", in core.js.

Here's how to reproduce the issue:

`import FacebookAPI from 'facebook-ads-sdk'` WITHOUT modification -> works fine.

----

```
cd node_modules/facebook-ads-sdk
npm install
Open core.js
Add 'console.log('hey') in the getNodePath() method
gulp dist
```

`import FacebookAPI from 'facebook-ads-sdk'` -> FacebookAPI becomes UNDEFINED.

I tried to import FacebookAPI at each step. The problem happens after the `gulp dist`.
Am I doing something wrong ?

Thank you,

**Comment by lucascosta** [comment]
I'll check it out and get back to you. Thank you

**Comment by lucascosta** [comment]
@DevJS, what environment are you in? Where are you putting this `import FacebookAPI from 'facebook-ads-sdk'`? If you are importing in ES6 you might have to import the class from the specific module or form the bundle, not the package itself. Are you compiling your code to ES5? If so the result would be like `var _facebookAdsSdk = require('facebook-ads-sdk');` which should contain the exported classes.

**Comment by lucascosta** [comment]
@DevJS did you manage to make it work?

**Comment by DevJS** [comment]
Hi sorry for the delay, had busy day at work.

So I tried to be sure that the problem was coming from the package and not from my environment.
I created a clean new project with basically only facebook-ads-sdk installed. 

The I simply made the modification specified in my first message and tried to compile the library again. And it worked! 

So I believe I have an issue with my environment. I'll try to figure this out and keep you updated. Thank you for your help 👍
